README.md

```diff
@@ -19,10 +20,83 @@ This is a project initiated by Mozilla to gather code coverage results on Firefo
 * [Minimum requirements](#minimum-requirements)
 * [License](#license)
 
-## Usage
 
-1. Download grcov from https://github.com/mozilla/grcov/releases or run ```cargo install grcov```
-2. Run grcov:
+## How to get a coverage report
```

Comment:
These are the instructions to use grcov to parse Rust coverage, but grcov is generic.
Can you keep the Usage section above as it was, and then rename this section `How to get a coverage report for Rust`

Reply:
I've added a little to show how one might use gcc or clang with grcov to hint that its generic.